### DISCUSSION TITLE
Controller HomeKit con luci Bticino Living Now

### DISCUSSION THREAD

**Discussion by ServiceA3** [indomus/forum#1736] [HUB: Home Assistant]
Buongiorno! gestisco le luci Living Now collegate al Gateway Controller HomeKit. Di punto in bianco sono andate offline! Qualcuno ha avuto lo stesso problema e mi sa dire come indagare la causa? Perchè nel log non c'è nulla..

**Comment by Salsalove** [comment]
A me è successo oggi...tutte le luci non disponibili. Tu hai risolto? @ServiceA3
